modular_skyrat/modules/interaction_menu/code/interaction_component.dm

```diff
@@ -212,9 +212,19 @@
 		return FALSE
 	if(!slot_index) // This condition is for the UI to decide if the button is shown at all. Slot index should never be null otherwise.
 		return TRUE
-	if(slot_index == ORGAN_SLOT_NIPPLES && !target.is_topless())
-		return FALSE
-	return target.is_bottomless()
+	switch(slot_index)
+		if(ORGAN_SLOT_NIPPLES)
+			var/tits_out = target.has_breasts(required_state = REQUIRE_GENITAL_EXPOSED)
+			if(!tits_out)
```

Comment:
you don't have to use a variable for this ((also you if you keep the variable, might want to use something more neutral for the variable name like `chest_exposed`))

Reply:
Very understandable, although we are trying to keep things clean on here.